bot/resource/cog/association/embed/asetin_embed.py

```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
 
     WHO_WE_ARE_FIELD_NAME: str = "Qui sommes-nous?"
     WHO_WE_ARE_FIELD_DESCRIPTION: str = (
-        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial."
+        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial.\n L'ASETIN représente les intérêts des étudiants et étudiantes de premier cycle en informatique auprès des différentes instances officielles de l’Université Laval, sur les différents comités et devant les autres associations étudiantes."
```

Comment:
Maybe no space between `\n` et `L'ASETIN`? See below :
With space: 
![image](https://github.com/user-attachments/assets/2c5b485a-1a9b-4757-bb6b-5d8368f053fc)


Without space:
![image](https://github.com/user-attachments/assets/c649849c-cf2a-470c-b244-d670267e4b65)

Reply:
Yeah I'll change that